### PR TITLE
Feature/add view for cart

### PR DESCRIPTION
**Pull request by DJCarstens**
Adds cart view
Adds functionality to populate cart view
Adds Login and logout functionality
re-organized some routing

### CODE REVIEW

**Review by RonzoP03 — APPROVED** [review]

File: WebStoreApplication/Controllers/APIControllers/OrdersController.cs [hunk 1]
```diff
@@ -33,13 +33,14 @@ public IActionResult GetOrder(int orderID)
...
         }
 
-        [HttpGet("{userID}")]
-        public IActionResult GetAllOrdersForUser(int userID)
+        [HttpGet("all/")]
+        public IActionResult GetAllOrderedItems()
         {
-            return Ok(dbAccessor.GetAllOrdersForUser(userID));
+            int userID = 2;
+            string statusDescription = "Accepted";
```
Comment (Vignesh-95) [inline WebStoreApplication/Controllers/APIControllers/OrdersController.cs:41]: Shouldn't userID and statusDescription be url parameters?

Comment (DJCarstens) [reply WebStoreApplication/Controllers/APIControllers/OrdersController.cs:41]: Ah I forgot this was in here. statusDescription will stay as is for now though until finalizations can be made with regards to the changing of statuses

Comment (DJCarstens) [reply WebStoreApplication/Controllers/APIControllers/OrdersController.cs:41]: Cart will always be the cart of the currently logged in user, that's why i get it from the session. statusDescription will stay hardcoded until changing statuses can be implemented or scrapped

Comment (Vignesh-95) [reply WebStoreApplication/Controllers/APIControllers/OrdersController.cs:41]: Oh I see. Cool Makes Sense.

**Review by Vignesh-95 — APPROVED** [review]

**Pull request merged**